AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py

```diff
@@ -150,41 +162,91 @@ def AtomEditorComponents_GlobalSkylightIBL_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.is_visible, global_skylight_entity.is_visible() is True)
 
-        # 8. Set the Diffuse Image asset on the Global Skylight (IBL) entity.
+        # 8. Set the default Diffuse Image asset on the Global Skylight (IBL) entity.
         diffuse_image_path = os.path.join("LightingPresets", "default_iblskyboxcm.exr.streamingimage")
         diffuse_image_asset = Asset.find_asset_by_path(diffuse_image_path, False)
         global_skylight_component.set_component_property_value(
             AtomComponentProperties.global_skylight('Diffuse Image'), diffuse_image_asset.id)
         Report.result(
-            Tests.diffuse_image_set,
+            Tests.default_diffuse_image_set,
             diffuse_image_asset.id == global_skylight_component.get_component_property_value(
                                           AtomComponentProperties.global_skylight('Diffuse Image')))
 
-        # 9. Set the Specular Image asset on the Global Light (IBL) entity.
+        # 9. Set the default Specular Image asset on the Global Light (IBL) entity.
         specular_image_path = os.path.join("LightingPresets", "default_iblskyboxcm.exr.streamingimage")
         specular_image_asset = Asset.find_asset_by_path(specular_image_path, False)
         global_skylight_component.set_component_property_value(
             AtomComponentProperties.global_skylight('Specular Image'), specular_image_asset.id)
         Report.result(
-            Tests.specular_image_set,
+            Tests.default_specular_image_set,
             specular_image_asset.id == global_skylight_component.get_component_property_value(
                                            AtomComponentProperties.global_skylight('Specular Image')))
 
-        # 10. Delete Global Skylight (IBL) entity.
+        #11. Set Exposure valuye to min value
+        global_skylight_component.set_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'), value=-5)
+        current_exposure_value = global_skylight_component.get_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'))
+        Report.result(Tests.exposure_set_max_value, current_exposure_value == -5)
```

Comment:
should this be `Tests.exposure_set_min_value`?

Reply:
Fixed!